tests/playbooks/test_minimal.yaml

```diff
@@ -31,6 +31,8 @@
     - neutron_adoption
     - glance_adoption
     - placement_adoption
+    - nova_adoption
     - cinder_adoption
     - horizon_adoption
     - dataplane_adoption
+    #- nova_compute_adoption
```

Comment:
Should nova compute adoption be distinct from the data plane adoption doc/role?

Reply:
please add it here also: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/playbooks/test_with_ceph.yaml#L27

Reply:
yes, I believe so, as the edpm adoption task should not ba caling nova specific bits?
We will need to apply migrations, and remove the FFU w/a from nova configs.
That should be done by that dedicated role, but I don't have a strong opinion.

@gibi WDYT?

Reply:
The data plane adoption currently is really just making sure that we can connect to the original node via EDPM. But i think the original intent was that the data plane adoption doc/test would get expanded to configure "all things data plane". We thought that just like greenfield deployment of EDP nodes typically happens "all at once, vertically integrated" using a single Deployment, adoption of EDP nodes would happen "all at once, vertically integrated" too.

I'm not saying it absolutely has to be that way, but generally i'm inclined to follow the greenfield pattern, unless there's something that would prevent us from doing that.

Reply:
updated edpm docs and tasks for FFU completion tasks